package.json

```diff
@@ -67,17 +67,9 @@
         "@azure/storage-blob": "^12.9.0",
         "@semantic-release/changelog": "^6.0.1",
         "@semantic-release/git": "^10.0.1",
-        "@storybook/addon-actions": "^7.6.17",
-        "@storybook/addon-controls": "^7.6.17",
-        "@storybook/addon-links": "^7.6.17",
-        "@storybook/addon-storysource": "^7.6.17",
-        "@storybook/addon-viewport": "^7.6.17",
-        "@storybook/addons": "^7.6.17",
-        "@storybook/cli": "^7.6.17",
-        "@storybook/components": "^7.6.17",
-        "@storybook/react": "^7.6.17",
-        "@storybook/react-webpack5": "^7.6.17",
-        "@storybook/theming": "^7.6.17",
+        "@storybook/addon-docs": "^10.1.9",
```

Comment:
The visualization of the color vars in the code panel are very shitty :(

<img width="1123" height="535" alt="image" src="https://github.com/user-attachments/assets/5e69c013-157c-48ab-a20e-04567174d8d9" />